src/install_pypi.rs

```diff
@@ -828,9 +828,30 @@ pub async fn update_python_distributions(
         let start = std::time::Instant::now();
 
         for dist_info in extraneous.iter().chain(reinstalls.iter()) {
-            let summary = uv_installer::uninstall(dist_info)
-                .await
-                .expect("uninstall did not work");
+            let summary = match uv_installer::uninstall(dist_info).await {
+                Ok(sum) => sum,
+                // Get error types from uv_installer
+                Err(UninstallError::Uninstall(e)) => {
```

Comment:
I don't know what werror type this is but maybe we can be even more specific on the uninstall error?

Reply:
Made it more specific